### PR TITLE
{RDBMS} Revert - Add MigrationRuntimeResourceId to properties of creating migration request in CLI

### PR DESCRIPTION
**Pull request by nasc17**
**Related command**
`az postgres flexible-server migration create --subscription sub --resource-group testGroup --name testserver --migration-name testmigration --properties "migrationConfig.json"`

**Description**
SDK may not get bumped up in time to include these changes in current milestone.
Revert #28695 

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
RDBMS

**Review by evelyn-ys — DISMISSED** [review]

File: src/azure-cli/azure/cli/command_modules/rdbms/_helptext_pg.py [hunk 1]
```diff
@@ -518,7 +518,7 @@
             },
             "SourceServerUserName": "testuser@pg-single-1",
             "TargetServerUserName": "fspguser"
-          },
+          }
```
Comment (ambrahma) [inline src/azure-cli/azure/cli/command_modules/rdbms/_helptext_pg.py:521]: Add missing commas

File: src/azure-cli/azure/cli/command_modules/rdbms/_helptext_pg.py [hunk 2]
```diff
@@ -543,7 +543,7 @@
...
```
Comment (ambrahma) [inline src/azure-cli/azure/cli/command_modules/rdbms/_helptext_pg.py:546]: same here

File: src/azure-cli/azure/cli/command_modules/rdbms/_helptext_pg.py [hunk 3]
```diff
@@ -574,7 +574,7 @@
             },
             "SourceServerUserName": "postgres",
             "TargetServerUserName": "fspguser"
-          },
+          }
```
Comment (ambrahma) [inline src/azure-cli/azure/cli/command_modules/rdbms/_helptext_pg.py:577]: missing commas

**Review by ambrahma — APPROVED** [review]

**Review by evelyn-ys — APPROVED** [review]

**Pull request merged**